web/packages/teleport/src/Assist/contexts/messages.tsx

```diff
@@ -222,12 +222,18 @@ async function convertServerMessage(
     });
 
     // The offset here is set base on A/B test that was run between me, myself and I.
-    const resp = await api
-      .fetch(sessionUrl + '/stream?offset=0&bytes=4096', {
-        Accept: 'text/plain',
-        'Content-Type': 'text/plain; charset=utf-8',
-      })
-      .then(response => response.text());
+    const resp = await api.fetch(sessionUrl + '/stream?offset=0&bytes=4096', {
+      Accept: 'text/plain',
+      'Content-Type': 'text/plain; charset=utf-8',
+    });
+
+    let msg;
+    let errorMsg;
+    if (resp.status === 200) {
+      msg = await resp.text();
+    } else {
+      errorMsg = 'Something went wrong';
```

Comment:
It probably should be "No session recording. The command execution failed" or something like that. Something went wrong suggest that something (fetching the log for example) failed. I also think that no session recording should return 4xx something.

Reply:
It returns a 403 forbidden for some reason I've updated the text